beginner_source/onnx/onnx_registry_tutorial.py

```diff
@@ -0,0 +1,451 @@
+# -*- coding: utf-8 -*-
+
+"""
+`Introduction to ONNX <intro_onnx.html>`_ ||
+`Export a PyTorch model to ONNX <export_simple_model_to_onnx_tutorial.html>`_ ||
+**Introduction to ONNX Registry**
+
+Introduction to ONNX Registry
+===========================
```

Comment:
match the number of `#` with the length of the string to prevent warningsin the doc build